config/jobs/kubernetes-sigs/controller-runtime/controller-runtime-presubmits-release-0.12.yaml

```
@@ -0,0 +1,34 @@
+presubmits:
+  kubernetes-sigs/controller-runtime:
+  - name: pull-controller-runtime-test-release-0-12
+    decorate: true
+    always_run: true
+    path_alias: sigs.k8s.io/controller-runtime
+    branches:
+    - ^release-0.12$
+    spec:
+      containers:
+      - image: golang:1.18
```

```suggestion
      - image: golang:1.18.1
```
I think this should be 1.18.1 to use the same version we used for main / to avoid the cert issue in go 1.18 here as well (same below)

---

done

---

I think we should not pin the minor version here so that we test it with the latest 1.18 version available always. 
Are we pinning for the other tests?

---

Oh you're right that makes more sense. Can you maybe change v1.18.1 to v1.18 in `controller-runtime-periodics-master.yaml` and `controller-runtime-presubmits-master.yaml`?

---

@sbueringer done :-) Good catch !

---

I introduced that issue :) Thx for fixing it